flink-formats/flink-parquet/src/main/java/org/apache/flink/formats/parquet/avro/AvroParquetReaders.java

```diff
@@ -46,7 +46,8 @@
      */
```

Comment:
Commit message: [FLINK-21406][parquet] Rename Parquet**_Avro_**Writers to AvroParquetWriters

Reply:
done